documentation/modules/metrics/con-metrics-cruise-control.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// metrics/assembly-metrics.adoc
+
+[id='con-metrics-cruise-control-{context}']
+= Monitoring Cruise Control operations
+
+[role="_abstract"]
+Kafka metrics are gathered by Cruise Control in order to track the utilization of brokers, topics and partitions.
+Cruise Control also provides a set of metrics for monitoring its own performance.
+
+The Cruise Control metrics reporter collects raw metrics data from Kafka brokers, which is produced to auto-created Cruise Control topics.
+The metrics are used to link:{BookURLUsing}#proc-generating-optimization-proposals-str[generate optimization proposals for Kafka clusters^].
+
+Cruise Control metrics are available for real-time monitoring of Cruise Control operations.
+For example, you can use metrics to monitor the status of rebalancing operations that are running or provide alerts on any anomalies that are detected in its performance.
```

Comment:
Are we now talking about the Strimzi Prometheus metrics? Is this included in the examples and Grafana dashboards? Or where is it available?

Reply:
This refers to the Cruise Control metrics scraped by Prometheus [1] We have a Grafana  dashboard here [2]

[1] https://github.com/linkedin/cruise-control/wiki/Sensors
[2] https://github.com/strimzi/strimzi-kafka-operator/blob/main/examples/metrics/grafana-dashboards/strimzi-cruise-control.json

Reply:
Changed to _For example, you can use Cruise Control_ metrics